mongodbatlas/resource_mongodbatlas_project.go

```diff
@@ -78,9 +79,10 @@ func resourceMongoDBAtlasProject() *schema.Resource {
 				Default:  true,
 			},
 			"api_keys": {
-				Type:     schema.TypeSet,
-				Optional: true,
-				Computed: true,
+				Type:       schema.TypeSet,
+				Optional:   true,
+				Computed:   true,
+				Deprecated: fmt.Sprintf(DeprecationMessageParameterToResource, "v1.12.0", "mongodbatlas_project_api_key"),
```

Comment:
The docs for project should also be updated with a deprecation message.

Reply:
Added

Reply:
Thank you!